dali/operators/decoder/audio/audio_decoder_op.h

```diff
@@ -0,0 +1,129 @@
+
+
+
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_OPERATORS_DECODER_AUDIO_AUDIO_DECODER_OP_H_
+#define DALI_OPERATORS_DECODER_AUDIO_AUDIO_DECODER_OP_H_
+
+#include <memory>
+#include <string>
+#include <vector>
+#include "dali/core/static_switch.h"
+#include "dali/operators/decoder/audio/audio_decoder.h"
+#include "dali/operators/decoder/audio/generic_decoder.h"
+#include "dali/pipeline/data/backend.h"
+#include "dali/pipeline/workspace/workspace.h"
+#include "dali/pipeline/operator/operator.h"
+#include "dali/pipeline/workspace/host_workspace.h"
+
+namespace dali {
+
+namespace detail {
+
+const std::string kOutputTypeName = "dtype";  // NOLINT
```

Comment:
```suggestion
const std::string kOutputTypeName = "output_dtype";  // NOLINT
```
I think this is used by most of the Operators, we should try to be consistent.

Reply:
Since we agreed on `dtype`, I'll keep this one, and proceed to figure out a way to change the rest of ops to have `dtype` as well